cmd/apptainer.lima

```diff
@@ -1,10 +1,17 @@
 #!/bin/sh
 set -eu
 : "${LIMA_INSTANCE:=apptainer}"
+: "${APPTAINER_BINDPATH:=}"
 
 if [ "$(limactl ls -q "$LIMA_INSTANCE" 2>/dev/null)" != "$LIMA_INSTANCE" ]; then
   echo "instance \"$LIMA_INSTANCE\" does not exist, run \`limactl start --name=$LIMA_INSTANCE template://apptainer\` to create a new instance" >&2
   exit 1
 fi
 export LIMA_INSTANCE
-exec lima APPTAINER_HOME="$HOME" apptainer "$@"
+if [ "$APPTAINER_BINDPATH" != "$HOME" ]; then
```

Comment:
I think this 'if' is unnecessary, but it doesn't hurt.

Reply:
I can remove it, it was for the corner case of alreadying being set

Reply:
`WARNING: While bind mounting '/home/anders:/home/anders': destination is already in the mount point list`

Reply:
But won't apptainer add that if $HOME is /home/anders regardless of whether or not it is in APPTAINER_BINDPATH?  By default it binds the home directory.  Who would have put it in APPTAINER_BINDPATH?

Reply:
That was if you did it explicitly, but since it was only a warning not an error (my bad) there is no need to watch out for it...